src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/UpdateVisitTest.kt

```diff
@@ -78,7 +78,7 @@ class UpdateVisitTest : IntegrationTestBase() {
     createApplicationAndSave(sessionTemplate = sessionTemplateDefault, completed = false, visitRestriction = visitRestriction)
 
     var expiredReservedApplication = applicationEntityHelper.create(slotDate = existingVisit.sessionSlot.slotDate, sessionTemplate = sessionTemplateDefault, completed = false, visitRestriction = visitRestriction)
-    applicationEntityHelper.createContact(application = expiredReservedApplication, name = "Jane Doe", phone = "01234 098765")
+    applicationEntityHelper.createContact(application = expiredReservedApplication, name = "Jane Doe", phone = "01234 098765", email = "example@email.com")
```

Comment:
should we replace all instances of example@email.com to email@example.com as email.com is an actual domain?

Reply:
yeah I can do this